tools/cli/helpers/promptForProject.js

```diff
@@ -69,3 +131,60 @@ export async function promptForType( options = { type: '' } ) {
 		type: options.type || typeAnswer.type,
 	};
 }
+
+/**
+ * Prompt for new project name.
+ *
+ * If no name is passed via `options`, then it will prompt for the name of project.
+ *
+ * @param {object} options - Passthrough of an object, meant to accept argv.
+ *
+ * @returns {object} object with the name property appended.
+ */
+export async function promptForName( options = { name: '' } ) {
+	let nameAnswer;
+
+	if ( ! options.name || options.name.length === 0 ) {
+		nameAnswer = await inquirer.prompt( {
+			type: 'input',
+			name: 'name',
+			message: 'What is your project called?',
+		} );
+	}
+
+	return {
+		...options,
+		name: options.name || nameAnswer.name,
+	};
+}
+
+/**
+ * Returns the appropriate list of questions.
+ *
+ * @param {string} type - The project type. Must be one of projectTypes
+ *
+ * @returns {Array} - Array of questions to ask.
+ */
+export function getQuestions( type ) {
```

Comment:
I'm assuming we may want to retrieve the questions from their own file, bit this'll do for now.

Reply:
also, why is this in this file vs command/generate?

Reply:
No reason other than "I don't know the best place to put things yet so you can live here for a minute" 😄

Reply:
I would move this to the generate command function. Helpers are, in my mind, things that will be reused across multiple commands. Anything singular command should live closer to it.